lib/cdm_utils.rb

```diff
@@ -3,10 +3,23 @@
 require "fileutils"
 
 module CDMUtils
+
+  def self.available_collections
+    digital_collections = DigitalCollection.all.map{ |c| c.collection_alias }
+  end
+
   def self.list(server)
```

Comment:
I'm not sure we need to call out to ContentDM at all here. We only want to see items that have active records, right? So doesn't it make sense to just ask active record directly, and skip the contentDM middle man? 
We coudl replace all of that with:
`DigitalCollection.pluck(:collection_alias, :name).to_h`
Am I missing some crucial aspect here?